content/src/service/caching/CacheManager.ts

```diff
@@ -68,4 +68,5 @@ export const ENTITIES_BY_POINTERS_CACHE_CONFIG = new CacheConfigBuilder('ENTITIE
   .withDefaultSize(EntityType.PROFILE, 2000)
```

Comment:
Please add a comment here to deprecate this, as it's not currently being used

Reply:
what should be deprecated? the `CacheManager` or the config?

Reply:
The config for the frequency of the snapshots per entity is not more being used, so maybe the `CacheManager` should be deprecated entirely, anyway I'm not sure if it's being used in another place. I want to add a comment here so we remember to deprecate this in the future.